scipy/spatial/_qhull.pyi

```diff
@@ -63,7 +63,7 @@ class _Qhull:
 
 def _get_barycentric_transforms(
     points: NDArray[np.float64],
-    simplices: NDArray[np.int_],
+    simplices: NDArray[np.intc],
```

Comment:
This is the correct fix here indeed. In the Cython code it's `np.ndarray[np.npy_int, ndim=2]`, and `npy_int` is defined in `numpy/__init__.pxd` as:
```C
ctypedef signed int       npy_int
```